bin/mozetl-submit.sh

```diff
@@ -82,7 +85,12 @@ fi
 pip install .
 python setup.py bdist_egg
 
-spark-submit --master ${MOZETL_SPARK_MASTER} \
-             --deploy-mode client \
-             --py-files dist/*.egg \
-             ${workdir}/runner.py ${MOZETL_ARGS}
+if [[ "${MOZETL_SUBMISSION_METHOD}" = "spark" ]]; then
+    spark-submit --master ${MOZETL_SPARK_MASTER} \
+                 --deploy-mode client \
+                 --py-files dist/*.egg \
+                 ${workdir}/runner.py ${MOZETL_ARGS}
+else
+    cd ${workdir}
```

Comment:
Does the egg need to be included in the path? What's the error that you get when you execute the runner?

Reply:
I guess I don't technically need the egg as we're in the source directory already.  Would you prefer consistency of using the egg in both branches of the bash script or just reduce the dependency on the find command?

Reply:
There is a `pip install .` that gets run earlier on. The egg is built so it can be distributed across the cluster for spark, but it might not be needed here.

Reply:
Ok, I've dropped the PYTHONPATH enviroment setting.